packages/api-clients/open-api/authorizationApi.yml

```diff
@@ -1833,23 +1833,23 @@ components:
       required:
         - name
         - members
-    ClientVisibility:
+    Visibility:
       type: string
-      enum: [COMPACT, FULL]
+      enum: [PARTIAL, FULL]
```

Comment:
Following suggestions from Alessio:
- I renamed ClientVisibility to Visibility, because it's generic enough to be reused for other resources, redefined in specs of different processes, etc. (for example, like we do with Problem or other schemas that we reuse around the monorepo)
- I renamed COMPACT to PARTIAL, because we already use compact with another meaning (not for visibility, but to say that we return just the data needed for that specific route/view)